presidio-anonymizer/presidio_anonymizer/anonymizer_engine.py

```diff
@@ -34,11 +34,14 @@ def anonymize(self, engine_request: AnonymizerRequest) -> str:
         ).to_sorted_unique_results(True)
         for analyzer_result in analyzer_results:
             transformation = engine_request.get_transformation(analyzer_result)
-            self.logger.debug(f"received transformation {transformation.get('type')}")
+            self.logger.debug(
+                f"for analyzer result {analyzer_result} received transformation "
+                f"{str(transformation)}")
             anonymizer_class = transformation.get("anonymizer")
             new_text = anonymizer_class().anonymize(transformation)
             end_of_text = min(analyzer_result.end, last_replacement_point)
             output_text = output_text[:analyzer_result.start] + new_text + output_text[
                                                                            end_of_text:]
             last_replacement_point = analyzer_result.start
+            self.logger.debug(f"text after replacement is {output_text}")
```

Comment:
Maybe original_text as well?

Reply:
Will be done once we add all the anonymizers. They require it. I don't want to add unneeded values for now...

Reply:
What do you mean? don't we have original_text on the top? We can just print it once for easy debug

Reply:
The original_text is the entire text is can be with 1,000 lines and more. 
I don't think it should be printed in debug.

Reply:
Do we write debug by default to app insights?

Reply:
Original text contains PII. We should be careful about logging it.

Reply:
In that case, when logging the output we might log PII which we failed to anonymize?

Reply:
Yes, could be.